mods/psionics/machines/psimonitor.dm

```diff
@@ -6,7 +6,7 @@
 	anchored = TRUE
 	density = TRUE
 	opacity = FALSE
-	req_access = list(list(access_psychiatrist, access_captain, access_cmo, access_hos))
+	initial_access = list(access_psychiatrist, access_captain, access_cmo, access_hos)
```

Comment:
keep the second list, you're changing the logic otherwise

Reply:
No, all network lists are OR based, not and based. This is the same functionality.

network_lock doesn't follow req_access spec.

Reply:
After discussing with afterthought, we decided to remove AND functionality completely from grant-building permission sets with network_lock. It seemed superfluous when you could alternatively just create more granular grants.